src/KlaviyoAPI.php

```diff
@@ -156,18 +152,25 @@ private function request( $method, $path, $options, $isPublic = false, $isV1 = f
     {
         $options = $this->prepareAuthentication( $options, $isPublic, $isV1 );
 
-        $response = $this->client->request( $method, $path, $options );
+        $setopt_array = $this->getDefaultCurlOptions($method);
```

Comment:
could we chain this so we don't set the same variable 3 times in a row?

Reply:
Yea, was going to go for something like this:
```
        $setopt_array = array_merge(
            $this->getDefaultCurlOptions($method),
            $this->getCurlOptUrl($path, $options),
            $this->getSpecificCurlOptions($options)
        );
```
However I realized merging arrays that use numeric keys (all those curl constants) for an associative array get reindexed :(. We can combine them and preserve their keys using "+"

Wish we had a nice python update method!